tsl/test/shared/sql/constify_now.sql.in

```diff
@@ -189,3 +189,47 @@ SELECT * FROM
   LATERAL (SELECT m2.time FROM devices LIMIT 1) as subq_1
 WHERE subq_1.time > m1.time;
 
+CREATE TABLE logged_data (
+    rawtag_id   INTEGER,
+    timestamp   TIMESTAMP WITH TIME ZONE NOT NULL,
+    value       REAL,
+    site_id     INTEGER
+);
+
+SELECT table_name FROM create_hypertable('logged_data', 'timestamp');
+
+INSERT INTO logged_data (rawtag_id, timestamp, value, site_id)
+VALUES
+(1, '2023-01-01'::timestamptz, 13, 1),
+(1, '2023-01-07'::timestamptz, 13, 1),
+(1, '2023-01-10'::timestamptz, 13, 1),
+(1, '2023-01-15'::timestamptz, 13, 1),
+(1, '2023-01-20'::timestamptz, 15, 1),
+(2, '2023-01-01'::timestamptz, 90, 1),
+(3, '2023-01-07'::timestamptz, 2, 1),
+(4, '2023-01-10'::timestamptz, 13, 3),
+(2, '2023-01-15'::timestamptz, 13, 4),
+(5, '2023-01-20'::timestamptz, 13, 1);
+
+-- four chunks, all of them should be excluded at plantime
+SELECT COUNT(*) FROM show_chunks('logged_data');
+
+SET timescaledb.current_timestamp_mock TO '2024-01-01 0:30:00+00';
+
+SET timescaledb.enable_chunk_append TO true;
+SET timescaledb.enable_constraint_aware_append TO true;
+
+-- no startup exclusion should be happening for any of the queries below
```

Comment:
It sounds like if the chunks shouldn't be excluded at all, might be more clear to say that they should be excluded at planning  time.